frontend/src/wrapper/SideNavigation.tsx

```diff
@@ -0,0 +1,233 @@
+import AdminIcon from '@mui/icons-material/AdminPanelSettings'
+import ChevronLeftIcon from '@mui/icons-material/ChevronLeft'
+import ContactSupportIcon from '@mui/icons-material/ContactSupport'
+import DashboardIcon from '@mui/icons-material/Dashboard'
+import LinkIcon from '@mui/icons-material/Link'
+import ListAltIcon from '@mui/icons-material/ListAlt'
+import SchemaIcon from '@mui/icons-material/Schema'
+import ViewList from '@mui/icons-material/ViewList'
+import {
+  Badge,
+  Divider,
+  IconButton,
+  List,
+  ListItem,
+  ListItemButton,
+  ListItemIcon,
+  ListItemText,
+  Toolbar,
+  Tooltip,
+} from '@mui/material'
+import MuiDrawer from '@mui/material/Drawer'
+import { styled, useTheme } from '@mui/material/styles'
+
+import { useGetNumApprovals } from '../../data/approvals'
+import { User } from '../../types/types'
+import Link from '../Link'
+
+const drawerWidth = 240
+
+const StyledList = styled(List)(({ theme }) => ({
+  paddingTop: 0,
+  paddingBottom: 0,
+  '&, & .MuiListItemIcon-root': {
+    color: theme.palette.primary.main,
+  },
+  '&& .Mui-selected, && .Mui-selected:hover': {
+    '&, & .MuiListItemIcon-root': {
+      color: theme.palette.secondary.main,
+    },
+  },
+}))
+
+const Drawer = styled(MuiDrawer, { shouldForwardProp: (prop) => prop !== 'open' })(({ theme, open }) => ({
+  '& .MuiDrawer-paper': {
+    position: 'relative',
+    whiteSpace: 'nowrap',
+    width: drawerWidth,
+    transition: theme.transitions.create('width', {
+      easing: theme.transitions.easing.sharp,
+      duration: theme.transitions.duration.enteringScreen,
+    }),
+    boxSizing: 'border-box',
+    ...(!open && {
+      overflowX: 'hidden',
+      transition: theme.transitions.create('width', {
+        easing: theme.transitions.easing.sharp,
+        duration: theme.transitions.duration.leavingScreen,
+      }),
+      width: theme.spacing(7),
+      [theme.breakpoints.up('sm')]: {
+        width: theme.spacing(9),
+      },
+    }),
+  },
+}))
+
+interface SideNavigationProps {
+  page: string
+  drawerOpen?: boolean
+  pageTopStyling?: any
```

Comment:
I think you can use the `CSSProperties` type provided by react: `import { CSSProperties } from 'react'